plugins/linear-genome-view/src/LinearGenomeView/components/Highlight.tsx

```diff
@@ -95,7 +96,16 @@ const Highlight = observer(function Highlight({ model }: { model: LGV }) {
       : undefined
   }
 
-  const h = mapCoords(model.highlight)
+  const asm = assemblyManager.assemblies.find(
```

Comment:
can use `assemblyManager.get(model.highlight.assemblyName)` instead of find here